### DISCUSSION TITLE
Welcome to DDlog Discussions!

### DISCUSSION THREAD

**Discussion by ryzhyk** [vmware/differential-datalog#861] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mihaibudiu** [comment]
Is this going to replace gitter?

**Reply by ryzhyk** [reply]
Hopefully!  One issue with gitter is that we cannot easily create new conversations there, so all discussions end up in the same thread. Here we can have any number of separate discussions.
